### PR TITLE
docs: Updating SETUP_SUBMITTER_CMF instructions to include steps for instal…

### PR DESCRIPTION
**Pull request by baxeaz**
…ling directly from pip or building a .whl file locally.

### What was the problem/requirement? (What/Why)
Setup instructions did not include steps for installing Python submitter dependencies from pip or building a whl from mainline and installing locally.

### What was the solution? (How)
Add setup instructions.

### What is the impact of this change?
Customers should better be able to manage their dependencies without submitter installer.

### How was this change tested?
Local testing

### Was this change documented?
It is only documentation.

### Is this a breaking change?
No
----

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: SETUP_SUBMITTER_CMF.md [hunk 1]
```diff
@@ -35,12 +35,13 @@ Deadline Cloud Monitor is used to both manage your credentials for submitting jo
 
 ## Deadline Software Installation
 
+- clone or download deadline-cloud-for-unreal-engine either from the release branch or mainline depending on whether you'd like the most recent tested release or all of the most recent commits.
```
Comment (joel-wong-aws) [inline SETUP_SUBMITTER_CMF.md]: nit: codeblock for `deadline-cloud-for-unreal-engine`?

File: SETUP_SUBMITTER_CMF.md [hunk 2]
```diff
@@ -35,12 +35,13 @@ Deadline Cloud Monitor is used to both manage your credentials for submitting jo
...
+
 ```
-python -m pip install deadline-cloud-for-unreal-engine
+git clone https://github.com/aws-deadline/deadline-cloud-for-unreal-engine.git
```
Comment (joel-wong-aws) [inline SETUP_SUBMITTER_CMF.md:41]: Is `cd deadline-cloud-for-unreal-engine.git` needed here?

File: SETUP_SUBMITTER_CMF.md [hunk 3]
```diff
@@ -54,9 +55,43 @@ runuat.bat BuildPlugin -plugin="C:\deadline\deadline-cloud-for-unreal-engine\src
...
+		}
+	]
+```
+
+Note that you may wish to disable the "strict hash" feature in Unreal's Python settings, or add hash settings for specific library and dependency versions you wish to consume.
+
+Finally if you're pulling from mainline you may have python dependencies which are not yet released to PyPi - you'll want to run something like this to build and install your mainline version.  Note that the .whl file will need to be changed to reflect the version which is output by hatch build:
+
+```
+pip install hatch (If not installed)
```
Comment (joel-wong-aws) [inline SETUP_SUBMITTER_CMF.md]: nit: can we move `(If not installed)` into a comment?

File: SETUP_SUBMITTER_CMF.md (hunk 2 above)
Comment (baxeaz) [reply SETUP_SUBMITTER_CMF.md:41]: It totally is yep!

File: SETUP_SUBMITTER_CMF.md (hunk 3 above)
Comment (baxeaz) [reply SETUP_SUBMITTER_CMF.md]: Sure!

File: SETUP_SUBMITTER_CMF.md (hunk 1 above)
Comment (baxeaz) [reply SETUP_SUBMITTER_CMF.md]: Hmm I have it in one below for the actual steps, is that what you meant?

Comment (joel-wong-aws) [reply SETUP_SUBMITTER_CMF.md]: Sorry, I was referring to having this sentence contain the `deadline-cloud-for-unreal-engine` surrounded by backticks to be consistent with other references to the name of the package.

Comment (baxeaz) [reply SETUP_SUBMITTER_CMF.md]: Ahh gotcha, sure!

File: SETUP_SUBMITTER_CMF.md [hunk 4]
```diff
@@ -54,9 +56,44 @@ runuat.bat BuildPlugin -plugin="C:\deadline\deadline-cloud-for-unreal-engine\src
 
 - Copy the “package” folder above to your Unreal installation’s Plugins folder (C:\Program Files\Epic Games\UE_5.4\Engine\Plugins\UnrealDeadlineCloudService for example)
 
-## Submitter Installer
+## Python Dependencies
+
+There are several ways to install the required Python dependencies.
+
+If you've built and installed the plugin from the release branch above, you can simply install from pip.  Either install as: (Adjust paths accordingly)
```
Comment (crowecawcaw) [inline SETUP_SUBMITTER_CMF.md]: Nit: I think it'd help to label or outline the 4 alternative methods to choose from in this section. Maybe adding a number to each alternative or giving each a label or header (e.g. "**Install from pip**: If you've build and installed...")

Comment (baxeaz) [reply SETUP_SUBMITTER_CMF.md]: Sure will add that

**Review by joel-wong-aws — APPROVED** [review]

**Review by crowecawcaw — APPROVED** [review]

**Pull request merged**